src/pages/Collection/index.tsx

```
@@ -378,49 +829,437 @@ export const Collection: React.FC = () => {
+            }}
+          />*/}
+          <PriceRangeContainer>
+            <Min>
+              <MinInputContainer className={isDarkTheme ? "dark" : "light"}>
+                <MinInputLabelContainer>
+                  <input
+                    placeholder="Min"
+                    onChange={(e) => {
+                      if (
+                        e.target.value === "" &&
+                        isNaN(parseInt(e.target.value))
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)